app/lib/features/chat_ng/widgets/chat_input/chat_editor.dart

```diff
@@ -153,20 +156,25 @@ class _ChatEditorState extends ConsumerState<ChatEditor> {
   }
 
   // chat editor UI
-  Widget renderEditorUI(bool emojiPickerVisible) {
-    return FrostEffect(
-      child: DecoratedBox(
-        decoration: BoxDecoration(
-          color: Theme.of(context).colorScheme.primaryContainer,
+  Widget renderEditorUI(bool emojiPickerVisible, bool? isEncrypted) {
+    return DecoratedBox(
+      decoration: BoxDecoration(
+        color: Theme.of(context).colorScheme.primaryContainer,
+        borderRadius: BorderRadius.only(
+          topLeft: Radius.circular(15.0),
+          topRight: Radius.circular(15.0),
         ),
-        child: Row(
-          children: [
-            leadingBtn(emojiPickerVisible),
-            editorField(),
-            trailingBtn(),
-          ],
+        border: BorderDirectional(
+          top: BorderSide(color: greyColor),
         ),
```

Comment:
I am not 100% sure about this new outline design change. It seems to be pushing the UI element to the front/above the other chat elements and thus giving the impression of being attached to the bottom bar rather than being part of the chat.

But let's not stop on this for now, we can revisit that once we are thinking of switching the labs to default on again.